doc/user/content/serve-results/s3.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Amazon S3"
+description: "How to export results from Materialize to Amazon S3."
+menu:
+  main:
+    parent: sink
+    name: "Amazon S3"
+    weight: 10
+---
+
+{{< private-preview />}}
+
+This guide walks you through the steps required to export results from
+Materialize to Amazon S3. Copying results to S3 is useful to perform tasks like
+periodic backups for auditing, or downstream processing in analytical data
+warehouses like [Snowflake](/serve-results/snowflake/), Databricks or
+BigQuery.
+
+## Before you begin
+
+- Ensure you have access to an AWS account, and permissions to create and manage
+  IAM policies and roles. If you're not an account administrator, you will need
+  support from one!
+
+## Step 1. Set up an Amazon S3 bucket
+
+First, you must set up an S3 bucket and give Materialize enough permissions to
+write files to it. We **strongly** recommend using [role assumption-based authentication](/sql/create-connection/#aws-permissions)
+to manage access to the target bucket.
+
+### Create a bucket
+
+1. Log in to your AWS account.
+
+1. Navigate to **AWS Services**, then **S3**.
+
+1. Create a new, general purpose S3 bucket with the suggested default
+   configurations.
+
+### Create an IAM policy
+
+Once you create an S3 bucket, you must associate it with an [IAM policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies.html)
+that specifies what actions can be performed on the bucket by the Materialize
+exporter role. For Materialize to be able to write data into the bucket, the
+IAM policy must allow the following actions:
+
+Action type  | Action name                                                                            | Action description
+-------------|----------------------------------------------------------------------------------------|---------------
+Write        | [`s3:PutObject`](https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutObject.html)      | Grants permission to add an object to a bucket.
+List         | [`s3:ListBucket`](https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjectsV2.html) | Grants permission to list some or all of the objects in a bucket.
+Delete        | [`s3:DeleteObject`](https://docs.aws.amazon.com/AmazonS3/latest/API/API_DeleteObject.html)| Grants permission to remove an object from a bucket.
```

Comment:
I accepted the suggestion, but in s3 this is actually listed under `Write`, so I'll revert.